src/orchard/clojuredocs.clj

```diff
@@ -103,7 +145,7 @@
   ([ns sym export-edn-url]
    (if (special-symbol? sym)
      (find-doc "clojure.core" (str sym) export-edn-url)
-     (-> (ns-resolve ns sym) var-name get-doc))))
+     (some-> (try-ns-resolve ns sym) var-name get-doc))))
```

Comment:
Why kind of exception are you anticipating here?

Reply:
This is the fix for https://github.com/clojure-emacs/cider-nrepl/blob/v0.22.3/src/cider/nrepl/middleware/clojuredocs.clj#L18